app/konnect/dev-portal/access-and-approval/dev-reg.md

```diff
@@ -20,16 +21,28 @@ All Developers must register through the {{site.konnect_short_name}} [Dev Portal
 
 2. At the bottom of the login form, click **Sign Up**.
 
-3. Fill out the registration form and click **Create Account**. 
+3. Fill out the registration form and click **Create Account**.
 
    You should receive an email to confirm your email address. Click the verification link in the email.
 
-5. Set a password for your account.
+4. Set a password for your account.
 
 If your admin has auto-approve enabled, you can log into the Dev Portal immediately after setting a password. Learn how to [set up auto-approval for developer access as an admin](/konnect/dev-portal/access-and-approval/auto-approve-devs-apps/).
 
 If auto-approve is not enabled, an admin has to review and approve your request manually. All manually approved developers are notified via email upon gaining access to the Dev Portal. Learn how to [manually approve developer access as an admin](/konnect/dev-portal/access-and-approval/manage-devs/#approve-dev-reg).
 
+<!-- markdownlint-disable-next-line -->
+## Login/Register with SSO {#sso}
+{:.badge .enterprise}
+
```

Comment:
Maybe we should add something like "Prequisitions - first OIDC needs to be setup within your organization"